worker-sys/src/ext/headers.rs

```diff
@@ -0,0 +1,20 @@
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
```

Comment:
Does this need to be buried in a `glue` module like some of the other extensions?

Reply:
Yes, I have included it in the glue module to match the others. Included in the glue module along with others